js/react_native/android/build.gradle

```
@@ -20,6 +20,24 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['OnnxruntimeModule_' + name]).toInteger()
 }
 
+def checkIfOrtExtensionsEnabled() {
+  // locate user's project dir
+  def reactnativeRootDir = project.rootDir.parentFile
+  // get package.json file in root directory
+  def packageJsonFile = new File(reactnativeRootDir, 'package.json')
+  // read field 'onnxruntimeExtensionsEnabled'
+  if (packageJsonFile.exists()) {
+    def packageJsonContents = packageJsonFile.getText()
+    def packageJson = new groovy.json.JsonSlurper().parseText(packageJsonContents)
+    return packageJson.onnxruntimeExtensionsEnabled == 'true'
+  } else {
+    throw new GradleException(
+        "Unable to find the package.json file in the project root directory."
+    )
+  }
+  return false
+}
+
```

consider storing the result of the check so we don't have to parse package.json multiple times
```suggestion

boolean ortExtensionsEnabled = checkIfOrtExtensionsEnabled()
```